common/persistence/src/main/java/app/coronawarn/server/common/persistence/repository/DiagnosisKeyRepository.java

```diff
@@ -69,4 +100,20 @@ void saveDoNothingOnConflict(
       @Param("rollingPeriod") int rollingPeriod,
       @Param("submissionTimestamp") long submissionTimestamp,
       @Param("transmissionRisk") int transmissionRisk);
+
+  @Modifying
+  @Query("INSERT INTO diagnosis_key "
+      + "(key_data, rolling_start_interval_number, rolling_period, submission_timestamp, transmission_risk_level, "
+        + "origin_country, visited_countries) "
+      + "VALUES (:keyData, :rollingStartIntervalNumber, :rollingPeriod, :submissionTimestamp, :transmissionRisk, "
+        + ":origin_country, :visited_countries) "
+      + "ON CONFLICT DO NOTHING")
+  void saveDoNothingOnConflictCountries(
```

Comment:
Might be a good thing  to completely replace the old saveDoNothingOnConflict with the new insert statement that includes countries